app/controllers/v1/courses_controller.rb

```diff
@@ -6,7 +6,8 @@ class CoursesController < ApplicationController
     def index
       render json: subject.courses.current_school_term,
              include: ['lesson_schedules', 'lesson_schedules.classroom',
-                       'lesson_schedules.classroom.building', 'subject'], status: :ok
+                       'lesson_schedules.classroom.building', 'subject',
+                       'enrolment_exist'], status: :ok
```

Comment:
Remove this, as we have the method, we can call it in the serializer. This is to include relationships